### PR TITLE
feat - Support glob input

### PR DESCRIPTION
**Pull request by bebraw**
Note that I extracted the logic to a worker. Unfortunately proselint isn't worker ready due to https://github.com/amperser/proselint/issues/313 .

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ npm install -g proselint
 ## Usage
 
 ```bash
-proselintjs <filename.md>
+proselintjs <glob>
```
Comment (sapegin) [inline Readme.md]: I’d prefer something like:

```
proselintjs <filename.md>
proselintjs <pattern/**/*.md>
```

File: index.js [hunk 2]
```diff
@@ -3,131 +3,58 @@
...
-};
+const glob = require('glob');
+const worker = require('./worker');
 
 /* ---------------- 8< -------- 8< ---------------- */
 
-const filepath = process.argv[2];
-if (!filepath) {
+const pattern = process.argv[2];
+if (!pattern) {
```
Comment (sapegin) [inline index.js:13]: What will it print when nothing found?

File: index.js [hunk 3]
```diff
@@ -3,131 +3,58 @@
...
+	const chunks = _.chunk(files, Object.keys(os.cpus()).length);
 
-	const lines = splitLines(contents);
-	remark()
-		.use(processCode)
-		.process(contents)
-	;
-	return lines.join('\n');
-}
+	const workers = workerFarm(
```
Comment (sapegin) [inline index.js]: Probably this can be removed.

**Review by sapegin — CHANGES_REQUESTED** [review]
Cool, thanks!

File: index.js (hunk 2 above)
Comment (bebraw) [reply index.js:13]: You'll get usage instructions. It would be possible to capture and show some message then, though.

Comment (sapegin) [reply index.js:13]: I mean when you pass a pattern / file name bug glob returns an empty array.

**Pull request merged**